travis/travis_install_nightly

```diff
@@ -15,22 +15,39 @@ fi
 
 : ${ODOO_REPO:="odoo/odoo"}  # default value, if not set
 IFS="/" read -a REPO <<< "${ODOO_REPO}"
-ODOO_URL="https://github.com/${REPO[0]}/${REPO[1]}/archive/${VERSION}.tar.gz"
-
-echo "Installing Odoo ${ODOO_URL}"
-wget -nv -O odoo.tar.gz ${ODOO_URL}
-tar -xf odoo.tar.gz -C ${HOME}
-
+ODOO_URL="https://github.com/${REPO[0]}/${REPO[1]}.git"
+ODOO_PATH="${HOME}/${REPO[1]}-${VERSION}"
+echo "==== Installing Odoo ${ODOO_URL} ===="
+git clone --depth=1 -b ${VERSION} ${ODOO_URL} ${ODOO_PATH}
+# wget -nv -O odoo.tar.gz ${ODOO_URL}
+# tar -xf odoo.tar.gz -C ${HOME}
+
+echo "==== Installing ooenv ===="
+# Patch for Odoo PR #6335
+cd ${ODOO_PATH}
+git apply ${TRAVIS_BUILD_DIR}/travis/0001-Backport-client-command-discovery-optimization.patch
+cd ${TRAVIS_BUILD_DIR}
+MQT_PATH=${HOME}/maintainer-quality-tools
+ln -s ${MQT_PATH}/ooenv/odoo-env ${ODOO_PATH}/addons/odoo-env
+
+echo "==== Installing Python dependencies ===="
 # Workaround to force using system site packages (see https://github.com/Shippable/support/issues/241#issuecomment-57947925)
 rm -f $VIRTUAL_ENV/lib/python2.7/no-global-site-packages.txt
-
 pip install -q -r $(dirname ${BASH_SOURCE[0]})/requirements.txt
 
 # Use reference .coveragerc
 cp ${HOME}/maintainer-quality-tools/cfg/.coveragerc .
 
-echo "Getting dependencies"
-clone_oca_dependencies
+echo "==== Getting dependencies ===="
+cd ${TRAVIS_BUILD_DIR}  # Maybe redundant, but just in case go into the build dir
+if [ "${VERSION}" -lt "8.0" ] ; then
+    clone_oca_dependencies
+else
+    ${ODOO_PATH}/odoo.py env . --index=${MQT_PATH}/ooenv/oca-index
+    echo "Possible modules:"
+    find . -maxdepth 1 -mindepth 1 -type d -printf '%f\n'
+    find . -maxdepth 1 -mindepth 1 -type d -printf '%f\n' | xargs ./odoo.sh get --verbose
```

Comment:
This runs the `get` command on every module in the repo to test. Of course the modules are already there, but their dependencies will still be processes, so this will trigger the download and availability of foreign modules we may depend on.